### ISSUE TITLE
Import from Google Authenticator app

### ISSUE THREAD

**Issue by nokje** [raivo-otp/ios-application#35]
The Google Authenticator app on iOS and Android provides the option to export the accounts via a QR code. You can export the account from 1 up to 5 simultanious accounts. This export method would greatly simplify moving from Google Authenticator to Raivo OTP.

https://support.google.com/accounts/answer/1066447?hl=en&co=GENIE.Platform=Android#zippy=%2Ctransfer-authenticator-codes-to-a-new-android-phone

**Comment by tijme** [comment]
I agree, this would be a very nice feature. It would be pretty straight forward to implement. The QR code consists of an `otpauth-migration://offline?data=` URI with Google Protocol Buffer data that can be parsed in Swift.

**Comment by vfxdev** [comment]
This would be an amazing feature that lots of people are looking for. Here's how Aegis has implemented it: https://github.com/beemdevelopment/Aegis/pull/406

**Comment by v6r** [comment]
bumpppp

**Comment by drlaserdev** [comment]
> I agree, this would be a very nice feature. It would be pretty straight forward to implement. The QR code consists of an `otpauth-migration://offline?data=` URI with Google Protocol Buffer data that can be parsed in Swift.

Has the feature been implemented then? Seems like an old issue thread that hasn't been closed.

**Comment by AldanTanneo** [comment]
Bump, this would be my requirement for switching from GAuth

**Comment by akkari93** [comment]
Came from a google search. Was super excited after downloading the app but now I'm stuck trying to move from G Authenticator

**Comment by zynth17** [comment]
any update on this? will this thread be open forever?

**Comment by Gertsch** [comment]
Bump! Please look into this. I honestly think you are losing people right a this step when trying to migrate.

**Comment by oviliz** [comment]
> Bump! Please look into this. I honestly think you are losing people right a this step when trying to migrate.

Indeed, just moved from Android to iOS and thought this might replace Aegis but not being able to import a json, text, html or scan the bulk QRs produced is absolute rubbish. Nobody wants to be locked with a vendor. Bye Raivo...
